obj/objects.h

```diff
@@ -68,9 +68,9 @@ namespace objects
     class EnergyCollector : public Sphere
     {
     public:
-        EnergyCollector(const core::Vec3 &origin, double energy) : _id(id), _energy(energy)
+        EnergyCollector(const core::Vec3 &origin, double energy = 0) : _energy(energy)
         {
-            setRadius(constants::kPi * constants::kSimulationRadius / constants::kPopulation);
+            setRadius(constants::kPi * (constants::kSimulationRadius / 2) / constants::kPopulation);
```

Comment:
Just checking. Is the radius should actually be multiplied by PI? Where is the PI coming from? Write it in the comment to explain why.

Reply:
yes, I will add comment maybe why it should be like this.

Reply:
Noted for deleting hard coded dependency